code/core/src/core-server/utils/StoryIndexGenerator.ts

```diff
@@ -506,7 +506,7 @@ export class StoryIndexGenerator {
     }
   }
 
-  chooseDuplicate(firstEntry: IndexEntry, secondEntry: IndexEntry): IndexEntry {
+  chooseDuplicate(firstEntry: IndexEntry, secondEntry: IndexEntry, projectTags: Tag[]): IndexEntry {
```

Comment:
Adjustment in `src/core-server/utils/StoryIndexGenerator.test.ts:1530:21` also necessary